src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -72989,28 +72989,249 @@ public void NullabilityOfTypeParameters_023()
             var source = @"
 class Outer
 {
-    void M<T>(dynamic? x)
+    void M1<T>(dynamic? x)
     {
         if (x == null) return;
```

Comment:
>x == null [](start = 12, length = 9)

Consider adding tests for ```!(x != null)```

Reply:
Sure, but since we don't know the behavior, we don't infer anything.

---
In reply to: [278750198](https://github.com/dotnet/roslyn/pull/34996#discussion_r278750198) [](ancestors = 278750198)